{"name": "Негативная проверка доверенного номера с символом '%'", "status": "passed", "description": "Проверка неудачного поиска клиента по доверенному номеру с символом '%'", "steps": [{"name": "Проверка сообщения об ошибке и кода ответа", "status": "passed", "start": 1716974673192, "stop": 1716974673193}], "parameters": [{"name": "proofNum", "value": "'996553565311%'"}], "start": 1716974673026, "stop": 1716974673193, "uuid": "f2bbd302-d976-4ca7-8b36-9e34382e0b91", "historyId": "8491f76211981660967405d3cd41fefe", "testCaseId": "a6be3475c37a4f6fbc7e20e79cd9ff1a", "fullName": "207_service_test.TestProofNumSuite#test_get_info_client_has_special_symbol_in_proofNum_get_error_response", "labels": [{"name": "subSuite", "value": "Проверки по доверенному номеру"}, {"name": "suite", "value": "(207 сервис) Получение информации по клиенту"}, {"name": "host", "value": "PC-411-140322D"}, {"name": "thread", "value": "27168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "207_service_test"}]}